src/averageAges.js

```diff
@@ -34,7 +47,19 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const allWomen = people.filter(elem => elem.sex === 'f');
+  const children = people.filter(mother =>
```

Comment:
it's better to name this variable 'mothers' cause this is what you're searching for

Reply:
that is not children, rename to mothers

Reply:
not fixed

Reply:
I meant @aliceblunt8's comment